src/encoded/static/components/genome_browser.js

```diff
@@ -126,6 +126,18 @@ const getDefaultCoordinates = (assembly, annotation, ignoreCache = false) => {
                 file_format: 'variant',
                 path: 'https://encoded-build.s3.amazonaws.com/browser/GRCh38/GRCh38-dbSNP153.vvariants-dir',
             },
+            {
+                file_format: 'bigBed',
```

Comment:
Why do you have this and another BigBed the ticket did not ask for? I see it in the picture in the image but no word in the ticket (unless I. missed it)

Reply:
> also should add rDHS track because it will make encyclopedia page cleaner.  This is really GRHCh38/mm10 specific, don’t care about lessor assemblies.

It wasn't in the ticket description, but we want both the "cCRE, all" and rDHS files for mm10 and GRCh38.